.circleci/config.yml

```diff
@@ -43,18 +43,23 @@ jobs:
     steps:
       - attach_workspace:
           at: ./
+      - run:
+          name: Set registry URL
+          command: npm set registry https://npm.jahia.com/
```

Comment:
I don't think this is needed ( looking at the logs it goes on the correct server), and it's probably better to specify a scope  :  `npm config set @jahia:registry https://npm.jahia.com/` (although setting the global registry should not break neither)